baybe/utils/memory.py

```diff
@@ -0,0 +1,62 @@
+"""Utilities for memory usage."""
+from typing import cast
+
+from baybe.parameters.base import DiscreteParameter, Parameter
+from baybe.utils.numerical import DTypeFloatNumpy
+
+
+def bytes_to_human_readable(num: float) -> tuple[str, str]:
+    """Turn float number representing memory byte size into a human-readable format.
+
+    Args:
+        num: The number representing a memory size in bytes.
+
+    Returns:
+        Tuple with the converted number string and its determined human-readable unit.
+    """
+    for unit in ["bytes", "KB", "MB", "GB", "TB", "PB", "EB", "ZB"]:
```

Comment:
I think we do not need all of these units :D

Reply:
at which completely arbitrarily decided unit do you wish to stop?